classes/controller/menu/class-postmeta-check-controller.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Post meta duplicate records check class
+ *
+ * @package P4BKS\Controllers\Menu
+ */
+
+namespace P4GBKS\Controllers\Menu;
+
+use P4GBKS\Command\Remove_Duplicate_Postmeta;
+
+if ( ! class_exists( 'Postmeta_Check_Controller' ) ) {
+
+	/**
+	 * Class Postmeta_Check_Controller
+	 */
+	class Postmeta_Check_Controller extends Controller {
+		/**
+		 * Create menu/submenu entry.
+		 */
+		public function create_admin_menu() {
+			if ( ! current_user_can( 'manage_options' ) ) {
+				return;
+			}
+
+			$current_user = wp_get_current_user();
+
+			if ( in_array( 'administrator', $current_user->roles, true ) && current_user_can( 'manage_options' ) ) {
+				add_submenu_page(
+					P4GBKS_PLUGIN_SLUG_NAME,
+					__( 'Postmeta Check', 'planet4-blocks-backend' ),
+					__( 'Postmeta Check', 'planet4-blocks-backend' ),
+					'manage_options',
+					'postmeta_report',
+					[ $this, 'postmeta_check' ]
+				);
+			}
+		}
+
+		/**
+		 * Handle form submit.
+		 *
+		 * @param mixed[] $data The form data.
+		 *
+		 * @return bool
+		 */
+		public function handle_submit( &$data ) : bool {
+			$remove_duplicate_postmeta = filter_input( INPUT_POST, 'delete_duplicate_postmeta', FILTER_SANITIZE_NUMBER_INT );
+			if ( 'POST' === $_SERVER['REQUEST_METHOD'] && $remove_duplicate_postmeta ) { //phpcs:ignore WordPress.Security.NonceVerification.Recommended
+				try {
+					$instance = new Remove_Duplicate_Postmeta();
+					try {
+						$is_deleted = $instance->remove_duplicate();
+					} catch ( \Exception $e ) {
+						$data['message'] = __( 'Exception: ', 'planet4-blocks-backend' ) . $e->getMessage();
+					}
+
+					if ( $is_deleted ) {
+						$data['message'] = __( 'Remove duplicate postmeta records successfully.', 'planet4-blocks-backend' );
+					} else {
+						$data['message'] = __( 'No duplicate postmeta record found.', 'planet4-blocks-backend' );
+					}
+				} catch ( \Error $e ) {
+					$data['message'] = $e->getMessage();
+				} catch ( \Exception $e ) {
+					$data['message'] = __( 'Exception: ', 'planet4-blocks-backend' ) . $e->getMessage();
+				}
+			}
+			return true;
```

Comment:
The return value here is not used and it's also always true. Probably makes more sense to not return anything here?